src/Sylius/Behat/Page/Shop/Account/Order/IndexPage.php

```diff
@@ -51,6 +51,11 @@ public function changePaymentMethod(OrderInterface $order): void
         $link->click();
     }
 
+    public function containsFlashMessage(string $message): bool
```

Comment:
```suggestion
    public function hasFlashMessage(string $message): bool
```
I don't force this change 🖖🏻.